cli/compose/loader/volume.go

```diff
@@ -70,6 +70,8 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 		switch option {
 		case "ro":
 			volume.ReadOnly = true
+		case "rw":
```

Comment:
Thinking of this; we'll probably need to add support for https://github.com/docker/docker/pull/31047 as well